<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="turngrad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>灰度</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetRaw" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widgetResult" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nearChange">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>24</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>临近插值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lineChange">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>24</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>线性插值</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
